windows-10-setup.md

```diff
@@ -7,45 +10,132 @@
-    - `sudo apt-get install build-essential python-is-python3 zsh`
-    - `chsh -s $(which zsh)`
+    - Once you have pasted it in, your Linux section should look like this (but instead of `maia`, it would say YOUR username):
+
+    ```js
+            {
+                "guid": "{07b52e3e-de2c-5db4-bd2d-ba144ed6c273}",
+                "hidden": false,
+                "name": "Ubuntu-20.04",
+                "source": "Windows.Terminal.Wsl",
+                "colorScheme": "One Half Dark",
+                "startingDirectory": "//wsl$/Ubuntu-20.04/home/maia"
+            },
+    ```
+1. Install some required libraries including zsh, make zsh the default shell, and restart your Windows Terminal
+    - Put this command into your terminal: `sudo apt-get install build-essential python-is-python3 zsh`
+    - Then this command: `chsh -s $(which zsh)`
     - Restart the Windows Terminal
+    - If you get a page full of info about "This is the Z Shell configuration for new users...", press q (Quit and do nothing)
 1. Install oh-my-zsh from inside the Windows Terminal
-    - `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
-    - Restart the terminal and open an Ubuntu tab
+    - Enter this command into your terminal (note that it's one long line, even if it displays as two lines on the page where you're reading this): `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
+    - Restart the terminal and open an Ubuntu tab with the little plus button in the top left corner
+    - If the prompt in your terminal window is now a little arrow and a tilde (~), instead of "yourname@...", that's OK (you'll change it again in a later step)
 1. Install VS Code if it isn't already installed
+    - https://code.visualstudio.com/download
 1. In your Ubuntu terminal, open VS Code with `code .`
 1. Install the following VS Code extensions
     - ESLint
-    - WSL - Remote
+    - Remote - WSL
     - Live Share (online students only)
     - vscode-icons (optional, but pretty :wink:)
     - Bracket Pair Colorizer (optional)
     - GitLens (optional)
 1. Restart your terminal
 1. Open VS Code again with `code .`
     - This should begin downloading the VS Code Server
-    - When prompted for access, accept it
+    - When prompted for access (by Windows Defender Firewall), click "Allow access"
 1. Install nvm
-    - `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash`
+    - Enter this command into your terminal: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash`
 1. Move the 3 nvm lines from the bottom of `.bashrc` to the bottom of `.zshrc`
+    - This is the trickiest step! 
+    - Run this command to open your .bashrc config file: `code ~/.bashrc`
+    - Scroll down to the bottom of the file and cut the three lines at the bottom that look like this: 
+
+    ```j
```

Comment:
I don't think you want this `j` after the 3 backticks. If anything, it should be `bash` or `sh`.

Reply:
Thanks for the pointer, I'll update it.